tests/driver_sx1280/Makefile

```diff
@@ -0,0 +1,24 @@
+BOARD ?= esp32-wroom-32
+
+include ../Makefile.tests_common
+
+# required modules
+USEMODULE += sx1280
+
+USEMODULE += shell
+USEMODULE += shell_commands
+
+USEMODULE += iolist
+
+ifeq ($(BOARD),esp32-wroom-32)
+# SPI CS
+CFLAGS += -DSX1280_PARAM_SPI_NSS=GPIO5
+# Optional RESET
+CFLAGS += -DSX1280_PARAM_RESET=GPIO32
+# Busy PIN
+CFLAGS += -DSX1280_PARAM_DIO0=GPIO26
+# IRQ
+CFLAGS += -DSX1280_PARAM_DIO1=GPIO33
+endif
```

Comment:
Would it make sense to add these as the default parameters and remove them from here? Local overrides for custom BOARDs should not be here IMO